fresh.sh

```
@@ -1,31 +1,28 @@
 LATESTPKG=`basename $LATESTURL`
 
-wget -c $LATESTURL
 
 VERSION=`echo $LATESTPKG | awk -F_ '{print $2}'`
 REVISION=`cat debian/revision | perl -ne 'chomp; print join(".", splice(@{[split/\./,$_]}, 0, -1), map {++$_} pop @{[split/\./,$_]}), "\n";'`
 
-echo XXXXXXXXXXXXXXXXXXXXXXXXXX Building $VERSION-$REVISION 
+echo XXXXXXXXXXXXXXXXXXXXXXXXXX Building $VERSION
 
-rm -rf tmp debian/data data
+wget -c $LATESTURL -O orig/flexibee_${VERSION}_all.deb
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)